cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialFunctionsTest.scala

```diff
@@ -61,31 +71,62 @@ class SpatialFunctionsTest extends DocumentingTest {
+          |* If the points are in the _WGS-84_ CRS category (2D), then the units of the returned distance will be meters, based on the haversine formula over a spherical earth approximation.
+          |* If the points are in the _WGS-84_ CRS category (3D), then the units of the returned distance will be meters.
+          |The distance is calculated in two steps.
+          |First, a haversine formula over a spherical earth is used, at the average height of the two points.
+          |To account for the difference in height, Pythagoras' theorem is used, combining the first calculated spherical distance and the height difference.
+          |This formula works well for points close to the earth's surface; for instance, it is well-suited for calculating the distance of an airplane flight.
+          |It is unsuitable for greater heights, however, such as when calculating the distance between two satellites.
         """.stripMargin)
-      function("distance(point1, point2)", "A Float.", ("point1", "A point in either the WGS 84 or cartesian CRS."), ("point2", "A point in the same CRS as 'point1'."))
-      considerations("`distance(null, null)`, `distance(null, point2)` and `distance(point1, null)` all return `null`.")
-      query("WITH point({x: 2.3, y: 4.5, crs: 'cartesian'}) as p1, point({x: 1.1, y: 5.4, crs: 'cartesian'}) as p2\nRETURN distance(p1,p2) AS dist", ResultAssertions((r) => {
-        r.toList.head("dist").asInstanceOf[Double] should equal(1.5)
-      })) {
-        p("The distance between two points in the _cartesian_ CRS is returned.")
+      function("distance(point1, point2)", "A Float.", ("point1", "A point in either the WGS 84 or Cartesian CRS category."), ("point2", "A point in the same CRS as 'point1'."))
+      considerations("`distance(null, null)`, `distance(null, point2)` and `distance(point1, null)` all return `null`.", "Attempting to use points with different Coordinate Reference Systems (such as WGS 84 2D and WGS 84 3D) will return `null`.")
+      query(
+        """WITH point({x: 2.3, y: 4.5, crs: 'cartesian'}) AS p1, point({x: 1.1, y: 5.4, crs: 'cartesian'}) AS p2
+          |RETURN distance(p1,p2) AS dist""".stripMargin, ResultAssertions((r) => {
+          r.toList.head("dist").asInstanceOf[Double] should equal(1.5)
+        })) {
+        p("The distance between two 2D points in the _Cartesian_ CRS category is returned.")
+        resultTable()
+      }
+      query(
+        """WITH point({longitude: 12.78, latitude: 56.7, height: 100}) as p1, point({latitude: 56.71, longitude: 12.79, height: 100}) as p2
+          |RETURN distance(p1,p2) as dist""".stripMargin, ResultAssertions((r) => {
+          Math.round(r.toList.head("dist").asInstanceOf[Double]) should equal(1270)
+        })) {
+        p("The distance between two 3D points in the _WGS 84_ CRS category is returned.")
```

Comment:
@petraselmer @craigtaverner 
I have been trying out the examples in this section with Cypher Shell, as Neo4j Browser doesn’t support the data type yet. I am having a few oddities but I am not sure if it is an error, and if so, where to report it. 

For example in the built manual under section _3.4.9.2. point() - WGS 84 2D_.
When I run:
`RETURN point({ longitude: 56.7, latitude: 12.78 }) AS point`

The manual says the result should be:

`Point{wgs-84, [56.7, 12.78]}`

However I am getting this:

`Point{srid=4326, x=56.7, y=12.78}`

Also, it says `Syntax: point({longitude | x, latitude | y [, crs]})`

and then further down in the table

`crs  |  The string 'WGS-84' `

This lead me to believe I could run the following:

`RETURN point({ longitude: 56.7, latitude: 12.78, 'WGS-84' }) AS point;`

But that gives an error.

When reading the test and the text more to the point (pun intended) I understand I should have done:

`RETURN point({ longitude: 56.7, latitude: 12.78, crs: 'WGS-84' }) AS point;`

This could be helped with an example. However I don’t think it is super important so I don’t insist.

Reply:
I don't think an example is needed... maybe just writing `'WGS-84'` would be an improvement over `The string 'WGS-84'` - so please change it accordingly.